mtprotoproxy.py

```diff
@@ -861,13 +864,82 @@ def set_instant_rst(sock):
     return False
 
 
+async def handle_proxy_protocol(reader, writer):
+    PROXY_SIGNATURE = b"\x50\x52\x4F\x58\x59\x20"
```

Comment:
b"PROXY " looks a bit nicer

Reply:
okay, both binary and literal definitions are allowed by protocol specs, so just matter of taste.
I'll change

Reply:
okay, both binary and literal definitions are allowed by protocol specs, so just matter of taste.
please note, it's b"PROXY " with trailing space to make sure subsequent split() will not rise runtime error. since b"PROXY " can be easily mis-trimmed by some innacurate "optimization" and "PROXY\x20" looks weird (imho), I've decided just to use binary formmat.
taking above into account, still need to change?